File: 01.py
#include <bits/stdc++.h>
using namespace std;

int main(){
    int e, v;
    cin >> e >> v;

    float h = e / (float)v;
    int H = e / v;
    int m = (h - H) * 60; 

    if(h + 19 < 24){
        cout << H + 19 << ":" << m << endl;
    }else{
        H = (H-5) %24;
        if(H <10){
            if(m >10){
                cout << "0" << H << ":" << m << endl;
            }
            else{
                cout << "0" << H << ":" << "0" << m << endl;
            }
        }
        else{
            if(m >10){
                cout << H << ":" << m << endl;
            }
            else{
                cout << H << ":" << "0" << m << endl;
            }
        }
    }
}

File: 01.cpp
#include <iostream>
#include <cstdlib>
#include <algorithm>

using namespace std;

int main(){
    int n; cin >> n;

    int list[n];{for(int i = 0; i < n; i++){list[i]= i+1;}}

    for(int i = 0; i < n; i++){cout << list[i] << " ";}{cout << "\n";}

    random_shuffle(list, list + n);

    for(int i = 0; i < n; i++){cout << list[i] << " ";}{cout << "\n";}
    return 0;
}


